bash/runcmdproc

```diff
@@ -23,17 +23,37 @@
 # debug flag; 0=no debug, 1=debug
 debug=0
 
-set -e              # exit on all errors
+#set -e              # exit on all errors
```

Comment:
Do you really want to never stop on errors.. I would prefer if you `set +e` when you want to handle return codes and then `set -e` again when you are finished.